week2/226_invert_binary_tree.py

```diff
@@ -0,0 +1,27 @@
+# https://leetcode.com/problems/invert-binary-tree/
+from typing import Optional
+
+
+class TreeNode:
+    def __init__(self, val=0, left=None, right=None):
+        self.val = val
+        self.left = left
+        self.right = right
+
+
+class Solution:
+    def invert_tree(self, root: Optional[TreeNode]) -> Optional[TreeNode]:
+        if not root:
+            return root
+
+        # DFS - Pre-Order traversal
+        # Post-Order also works - is it faster than Pre-Order?
+        # In-Order does not work because the root nodes swaps sides again
+        # after the left descendant were swapped
+        temp = root.left
```

Comment:
In python, you can do
`root.left, root.right = root.right, root.left` 
This would be considered to be more [pythonic](https://www.udacity.com/blog/2020/09/what-is-pythonic-style.html) :)